amd/src/embed.js

```diff
@@ -22,7 +22,10 @@ define(['jquery', 'mod_hvp/communicator'], function($, H5PEmbedCommunicator) {
                 var resizeDelay;
                 var instance = H5P.instances[0];
                 var parentIsFriendly = false;
-
+                var completionactivity = document.getElementsByClassName('activity-header')[0];
```

Comment:
Are we calculating the `completionactivity` regardless of if completion is enabled or not?

Reply:
I have added a control on the completion state here